chains/centrifuge/connection.go

```diff
@@ -12,6 +14,8 @@ import (
 
 var _ chains.Connection = &Connection{}
 
+var MetaLock = &sync.RWMutex{}
```

Comment:
Please make this a field on `Connection`. It's possible we have multiple substrate chains running